sonar-project.properties

```diff
@@ -1,5 +1,6 @@
 # Project information
-sonar.projectKey=eatme-global_react-native-pos-thermal-printer_1c82270c-872d-4016-beab-c2d94c46c490sonar.projectName=react-native-esc-pos-printer
+sonar.projectKey=eatme-global_react-native-pos-thermal-printer_1c82270c-872d-4016-beab-c2d94c46c490
+sonar.projectName=react-native-esc-pos-printer
```

Comment:
project name should be the same as the repo name. Refer to pos-app configuration and update everything.

Reply:
Sure